### PR TITLE
New package: Panda3D.Panda3D version 1.10.14

### LINKED ISSUE #155232: [Package Request]: Panda3D
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

_Download Page Url:_ **https://www.panda3d.org/download/sdk-1-10-14/**
_Publisher:_ **Carnegie Mellon University**
_Package Name:_ **Panda3D**
_Description:_ **Powerful, mature open-source cross-platform game engine for Python and C++, developed by Disney and CMU**
_Package Version:_ **1.10.14**
_Installer URL:_ **https://github.com/panda3d/panda3d/releases**

### PR DESCRIPTION
**Pull request by schembriaiden**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.2.1 :rocket:

Resolves: https://github.com/microsoft/winget-pkgs/issues/155232
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/155242)

### CODE REVIEW

File: manifests/p/Panda3D/Panda3D/1.10.14/Panda3D.Panda3D.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Panda3D.Panda3D
+PackageVersion: 1.10.14
+PackageLocale: en-US
+Publisher: Carnegie Mellon University
+PublisherUrl: https://www.panda3d.org/
+PackageName: Panda3D
```
Comment (Trenly) [inline manifests/p/Panda3D/Panda3D/1.10.14/Panda3D.Panda3D.locale.en-US.yaml:9]:
```suggestion
PackageName: Panda3D SDK
```

File: manifests/p/Panda3D/Panda3D/1.10.14/Panda3D.Panda3D.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Panda3D.Panda3D
+PackageVersion: 1.10.14
+PackageLocale: en-US
+Publisher: Carnegie Mellon University
+PublisherUrl: https://www.panda3d.org/
+PackageName: Panda3D
+PackageUrl: https://www.panda3d.org/
+License: Freeware
```
Comment (Trenly) [inline manifests/p/Panda3D/Panda3D/1.10.14/Panda3D.Panda3D.locale.en-US.yaml:11]:
```suggestion
License: Modified BSD
```

**Review by Trenly — CHANGES_REQUESTED** [review]
This package doesn't write a `DisplayVersion`, but does write the version as part of the product code and `DisplayName` - please add `AppsAndFeaturesEntries`

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Review by stephengillie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**